src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -56,6 +56,7 @@ const AnnotationActivity = ({
 
     const handleEdit = () => {
         setIsEditing(true);
+        setIsInputOpen(true);
```

Comment:
Both `isEditing` and `isInputOpen` will be set to true if the user clicks on Modify button.